boards/arm/nrf5340_audio_dk_nrf5340/Kconfig.defconfig

```diff
@@ -0,0 +1,78 @@
+# nRF5340 Audio DK board configuration
+
+# Copyright (c) 2019 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP || BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP_NS
+
+config BOARD
+	default "nrf5340_audio_nrf5340_cpuapp" if (BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP || BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP_NS)
```

Comment:
What's the point in repeating this condition here?

Reply:
The if on line 9 has been removed